backend/src/main/java/com/bakdata/conquery/models/query/concept/TableExportQuery.java

```diff
@@ -107,19 +116,91 @@ public void collectRequiredQueries(Set<ManagedExecution> requiredQueries) {
 	@Override
 	public void resolve(QueryResolveContext context) {
 		query.resolve(context);
-		resolvedHeader = new ArrayList<>();
+
+		AtomicInteger currentPosition = new AtomicInteger(1); // First is dates
```

Comment:
Das kann auch ein normaler int sein

Reply:
Ohne validityDateColumn gehts dann trotzdem bei 1 los?

Reply:
Das ist dann leer. Sehe ich nicht so als Problem, da wir per default immer eins nehmen.

Reply:
> Das kann auch ein normaler int sein

Nein, weil wir es in den Stream Operationen und lambdas benutzen.